<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerWidget</class>
 <widget class="QWidget" name="CustomerWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/customer.svg</normaloff>:/icons/customer.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="documentLabel">
       <property name="text">
        <string>Document:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="documentLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="addressLine1LineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="addressLine1Label">
       <property name="text">
        <string>Address Line 1:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="addressLine2Label">
       <property name="text">
        <string>Address Line 2:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="addressLine2LineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="zipCodeLabel">
       <property name="text">
        <string>Zip Code:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="zipCodeLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="birthdateLabel">
       <property name="text">
        <string>Birthdate:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateEdit" name="birthdateDateEdit">
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2100</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>1910</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Country:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="countryComboBox"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="stateLabel">
       <property name="text">
        <string>State:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="stateComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="cityLabel">
       <property name="text">
        <string>City:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="cityComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="emailLabel">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="emailLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>documentLineEdit</tabstop>
  <tabstop>addressLine1LineEdit</tabstop>
  <tabstop>addressLine2LineEdit</tabstop>
  <tabstop>zipCodeLineEdit</tabstop>
  <tabstop>emailLineEdit</tabstop>
  <tabstop>birthdateDateEdit</tabstop>
  <tabstop>countryComboBox</tabstop>
  <tabstop>stateComboBox</tabstop>
  <tabstop>cityComboBox</tabstop>
  <tabstop>saveBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>countryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CustomerWidget</receiver>
   <slot>setUpStatesModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stateComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CustomerWidget</receiver>
   <slot>setUpCitiesModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>CustomerWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setUpStatesModel()</slot>
  <slot>setUpCitiesModel()</slot>
  <slot>save()</slot>
 </slots>
</ui>
